google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -357,6 +359,15 @@ def clean_up_spanner_objects
   puts "Error while cleaning up instances and databases after spanner tests.\n\n#{e}"
 end
 
+def cleanup_all_databases(instance)
+  puts "Cleaning up all databases."
```

Comment:
Is if safe to do this operation? Do we have a database name convention so that we only delete databases related to this test?

Reply:
Or only delete `stale` databases, e.g., was created a day ago.

Reply:
Yes, updated code.